rules/gcp_audit_rules/gcp_firewall_rule_modified.py

```diff
@@ -0,0 +1,21 @@
+import re
+
+from gcp_base_helpers import gcp_alert_context
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    method_pattern = r"(?:\w+\.)*v1\.(?:Firewall\.Update)|(compute\.firewalls\.(patch|update))"
+    match = re.search(method_pattern, deep_get(event, "protoPayload", "methodName", default=""))
+    return match is not None
+
+
+def title(event):
+    actor = deep_get(event, "protoPayload", "authenticationInfo", "principalEmail", default="")
+    method = deep_get(event, "protoPayload", "methodName", default="")
+    resource = deep_get(event, "protoPayload", "resourceName", default="")
```

Comment:
```suggestion
    actor = deep_get(event, "protoPayload", "authenticationInfo", "principalEmail", default="<ACTOR_NOT_FOUND>")
    method = deep_get(event, "protoPayload", "methodName", default="<METHOD_NOT_FOUND>")
    resource = deep_get(event, "protoPayload", "resourceName", default="<RESOURCE_NOT_FOUND>")
```

Reply:
Addressed in [`b5e3a30` (#785)](https://github.com/panther-labs/panther-analysis/pull/785/commits/b5e3a30a1a98e5317b5aac1664acca5b223bd2f4).

I also updated my rule from yesterday.